integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisApplicationIntegrationTest.java

```diff
@@ -93,6 +93,18 @@
 import org.junit.jupiter.api.TestInfo;
 import org.junit.jupiter.api.extension.ExtendWith;
 
+/**
+ * @implSpec This test expects the server to be configured with the following features enabled:
```

Comment:
WDYT about putting these notes in a README under `integration-test` and mention the README in javadoc?